### PR TITLE
Admin: Ensure the default value has been selected 

### PR DESCRIPTION
**Pull request by Pikkupomo**
Admin: Ensure the default value has been selected
* In Picotable, if there was no default given, the filter dropdowns had no content. This commit adds the default value there selected by default.

Tasks: Add link to the task comment author

### CODE REVIEW

File: shuup/tasks/templates/shuup/admin/tasks/task_edit.jinja [hunk 1]
```diff
@@ -44,9 +44,10 @@
                                     </thead>
                                     <tbody>
                                         {% for comment in comments %}
+                                        {% set url = shuup_admin.model_url(comment.author) %}
                                         <tr>
                                             <td class="col-xs-2">{{ comment.created_on|datetime(format="short") }}</td>
-                                            <td class="col-xs-2">{{ comment.author }}</td>
+                                            <td class="col-xs-2"><a href="{{ url }}">{{ comment.author.id }}</a></td>
```
Comment (chessbr) [inline shuup/tasks/templates/shuup/admin/tasks/task_edit.jinja]: is there any purpose to show the customer id instead of the name?

File: shuup/tasks/locale/en/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -17,156 +17,160 @@ msgstr ""
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 
+#: admin_module/__init__.py:26 admin_module/__init__.py:53
```
Comment (chessbr) [inline shuup/tasks/locale/en/LC_MESSAGES/django.po]: why all these file references?

Comment (Pikkupomo) [reply shuup/tasks/locale/en/LC_MESSAGES/django.po]: wrong makemessages :)

File: shuup/tasks/templates/shuup/admin/tasks/task_edit.jinja (hunk 1 above)
Comment (Pikkupomo) [reply shuup/tasks/templates/shuup/admin/tasks/task_edit.jinja]: yes, register from front -> go to dashboard -> request data anonymization. 

TL;DR; not all contacts have name :)

I can change it so that if name exists, we show that and if not, we show id (like on the Picotable this PR)
